processor/metricstransformprocessor/config.go

```diff
@@ -80,7 +83,8 @@ type Operation struct {
 	// AggregatedValues is a list of label values to aggregate away.
 	AggregatedValues []string `mapstructure:"aggregated_values"`
 
-	// NewValue indicates what is the value called when the AggregatedValues are aggregated into one.
+	// NewValue indicates what is the value called when the AggregatedValues
+	// are aggregated into one or a new label is added to an existing metric.
 	NewValue string `mapstructure:"new_value"`
```

Comment:
Side note: we probably need to break this struct up for different operation types at some point.

Reply:
This is just an error in setting the label values at the metric descriptor and not at the time series. Since the intent of hte pr is to update the label values they have the same goal (either in add a new label or when it gets aggregated). To make it cleared, i'll update the comment to say
```
// NewValue is used to set a new label value either when the operation is `AggregatedValues` or `AddLabel`.
```